L/libseat/build_tarballs.jl

```diff
@@ -0,0 +1,49 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "libseat"
```

Comment:
Upstream project seems to be called `seatd`?

Reply:
Yeah, let's go with that name instead